### DISCUSSION TITLE
Wyze Bluetooth bolt lock

### DISCUSSION THREAD

**Discussion by Shannondalebreaux** [SecKatie/ha-wyzeapi#414] [General]
![Screenshot_20220929-120744_Brave](https://user-images.githubusercontent.com/62623580/193094764-35411079-894f-4d51-a3c1-ee1df606bb0b.jpg)
Can home assistant work with this lock ? Wyze Bluetooth bolt lock, If not is there away to reverse engineer so it can work with it? 
https://www.wyze.com/products/wyze-lock-bolt?variant=41693814784162&currency=USD&utm_medium=product_sync&utm_source=google&utm_content=sag_organic&utm_campaign=sag_organic&utm_campaign=gs-2022-03-17&utm_source=google&utm_medium=smart_campaign

**Comment by JoeSchubert** [comment]
There's basically no way to make something that is Bluetooth only work with this integration without writing some sort of local relay that would get commands from the integration and relay them via a Bluetooth device that is within range of the bolt.
